code/nodes/featureSignificance/funcs.q

```diff
@@ -0,0 +1,59 @@
+\d .automl
+
+// Definitions of the main callable functions used in the application of .automl.featureSignificance
+
+// @kind function
+// @category featureSignificance
+// @fileoverview Extract feature significant tests and apply to feature data
+// @param cfg   {dict} Configuration information assigned by the user and related to the current run
+// @param feats {tab}  The feature data as a table 
+// @param tgt   {(num[];sym[])} Numerical or symbol vector containing the target dataset
+// @return {sym[]} Significant features or error if function does not exist
+featureSignificance.applySigFunc:{[cfg;feats;tgt]
+  sigFunc:util.qpyFuncSearch cfg`sigFeats;
+  sigFunc[feats;tgt]
+  }
+
+// @kind function
+// @category featureSignificance
+// @fileoverview Apply feature significance function to data post feature extraction
+// @param cfg   {dict}          Configuration information assigned by the user and related to the current run
+// @param feats {tab}           The feature data as a table 
+// @param tgt   {(num[];sym[])} Numerical or symbol vector containing the target dataset
```

Comment:
This can only be a numeric list, at this point the data will have been encoded if symbol